src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -1440,10 +1441,7 @@ public boolean isGoodHttpStatus(int status) {
         if ((400 <= status) && (status <= 417)) {
             return false;
         }
-        if ((500 <= status) && (status <= 505)) {
-            return false;
-        }
-        return true;
+        return (500 > status) || (status > 505);
```

Comment:
Reduces legibility in this case (parallel with previous clauses destroyed), but whatever.

Reply:
Both reverted to if statements.